src/internet_identity/src/main.rs

```diff
@@ -563,7 +564,8 @@ mod v2_api {
     #[update]
     #[candid_method]
     fn identity_info(identity_number: IdentityNumber) -> Result<IdentityInfo, ()> {
-        authenticate_and_record_activity(identity_number);
+        authenticate_and_record_activity(identity_number)
```

Comment:
Given this is API v2, this error will be propagated to the client in an upcoming PR.